numpy/core/arrayprint.py

```diff
@@ -479,9 +480,11 @@ def array2string(a, max_line_width=None, precision=None,
     prefix : str, optional
         An array is typically printed as::
 
-          'prefix(' + array2string(a) + ')'
+          prefix + array2string(a) + suffix
 
         The length of the prefix string is used to align the output correctly.
```

Comment:
_"The length of the prefix and suffix strings are used to respectively align and wrap the output"_, maybe?